CONTRIBUTING.rst

```diff
@@ -474,7 +474,7 @@ created the following will occur:
 
     .. code:: bash
 
-       git tag MAJOR.MINOR.0
+       git tag v$(python -c "import pyvista as pv; print(pv.__version__)")
```

Comment:
Great idea! This would have prevented the `v0.35.0` release issue I had a few months back.

Reply:
Hm. We have a mismatch of tags with the `v` prefix and not. Seems this was adopted at v0.33.0? Is this intentional? I generally do not like to use the `v` prefix

Reply:
Otherwise, this is a nice, clean way to set the tag. Definitely will adopt elsewhere too

Reply:
I felt the same way, but the main problem of not adopting `v*` as a tag prefix is it makes it difficult to either `grep` or filter tags. Additionally, with github's new [tag proection rules](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-tag-protection-rules), without a prefix we can't restrict the tags we want to protect.

So far, the last several tags have been:
```
0.30.0
0.30.1
0.31.0
0.31.1
0.31.2
0.31.3
0.32.0
0.32.1
v0.33.0
v0.33.1
v0.33.2
v0.33.3
v0.34.0
v0.34.1
```
So, we've defacto switched over. We can change back, but I think I see the logic in prefixing tags now.